src/lib/index.js

```diff
@@ -246,6 +246,10 @@ export const getStateFromUrlParams = (defaultState = {}) => {
           return { id: k, desc: v === 'desc' };
```

Comment:
I am a bit lost. Ithought we kept `getStateFromUrlParams`for legacy purposes ? If it is the case we shouldnot need to implement `,` as past urlparams should not use it ?